driver/src/price_finding/naive_solver.rs

```diff
@@ -112,11 +112,7 @@ impl NaiveSolver {
 }
 
 impl PriceFinding for NaiveSolver {
-    fn find_prices(
-        &self,
-        orders: &[Order],
-        state: &AccountState,
-    ) -> Result<Solution, PriceFindingError> {
+    fn find_prices(&self, orders: &[Order], state: &AccountState) -> Result<Solution> {
```

Comment:
So we no longer include Error as part of the `Result`?

Reply:
The `anyhow` has a result type alias like this: `type Result<T> = std::result::Result<T, anyhow::Error>`, it basically means all our results use the same error type.